### PR TITLE
#157966002 Setup frond-end client

### PR DESCRIPTION
**Pull request by chukwuemekachm**
#### What does this PR do?
- Completes front-end client
#### Description of Task to be completed?
- Add public folder in project to serve static files
- Add index.html to public folder
- Add js and css folders to public folder

#### How should this be manually tested?
- Clone the repo
- Startup command line
- cd to project root
- run ``` npm install ```
- run ``` npm run start:dev ```
- Open your browser and navigate to `localhost:3000/client`
#### What are the relevant pivotal tracker stories?
- #157966002
#### Screenshots (if appropriate)

screenshot 195

#### Questions:
- None

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
 module.exports = {
-    "extends": "airbnb"
+    "extends": "airbnb",
+	"globals": {
```
Comment (benfluleck) [inline .eslintrc.js:3]: This indentation is off

File: public/css/Poiret_One/OFL.txt [hunk 2]
```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2011, Denis Masharov (denis.masharov@gmail.com)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+http://scripts.sil.org/OFL
+
+
+-----------------------------------------------------------
```
Comment (benfluleck) [inline public/css/Poiret_One/OFL.txt:8]: What is this file doing here please remove, import fonts using CDNs

File: public/css/Tajawal/OFL.txt [hunk 3]
```diff
@@ -0,0 +1,93 @@
+Copyright 2018 Boutros International. (http://www.boutrosfonts.com)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
+http://scripts.sil.org/OFL
+
+
```
Comment (benfluleck) [inline public/css/Tajawal/OFL.txt:7]: same as above ?

**Comment by chukwuemekachm** [comment]
I've done that

**Pull request merged**